aea/protocols/oef/message.py

```diff
@@ -53,12 +51,10 @@ class OEFErrorOperation(Enum):
 
         REGISTER_SERVICE = 0
         UNREGISTER_SERVICE = 1
-        REGISTER_DESCRIPTION = 2
```

Comment:
Are we removing REGISTER_DESCRIPTION and UNREGISTER_DESCRIPTION because they are duplicating the SERVICE_ equivalents?

Reply:
No, because they are related to the `register_agent` and `unregister_agent`, which have been removed.

Reply:
yes